examples/hello/hello.cpp

```diff
@@ -243,7 +237,10 @@ int initialize()
     Renderer::Desc rendererDesc;
     rendererDesc.width = gWindowWidth;
     rendererDesc.height = gWindowHeight;
-    gRenderer->initialize(rendererDesc, getPlatformWindowHandle(gWindow));
+    {
+        const SlangResult res = gRenderer->initialize(rendererDesc, getPlatformWindowHandle(gWindow));
```

Comment:
Note: this is exposing the fact that our API wrapper layer which is "totally not part of Slang" is using the Slang result codes, etc. I'm not entirely sure how I feel about that.